app/services/katello/pulp/repository.rb

```diff
@@ -90,8 +90,12 @@ def sync(overrides = {})
       end
 
       def create
-        smart_proxy.pulp_api.extensions.repository.create_with_importer_and_distributors(repo.pulp_id, generate_importer,
-                                                                              generate_distributors, display_name: root.name)
+        importer = generate_importer
+        smart_proxy.pulp_api.extensions.repository.create_with_importer_and_distributors(
+          repo.pulp_id,
+          importer,
```

Comment:
any reason to not just call generate_importer here?  (wondering if this was from leftover debugging)

Reply:
yes, I was spelunking!